erpnext/accounts/report/sales_register_by_item/sales_register_by_item.js

```diff
@@ -35,23 +49,11 @@ frappe.query_reports["Item-wise Sales Register"] = {
 			"fieldtype": "Link",
 			"options": "Warehouse"
 		},
-		{
-			"fieldname": "brand",
-			"label": __("Brand"),
-			"fieldtype": "Link",
-			"options": "Brand"
-		},
-		{
-			"fieldname": "item_group",
-			"label": __("Item Group"),
-			"fieldtype": "Link",
-			"options": "Item Group"
-		},
 		{
 			"label": __("Group By"),
 			"fieldname": "group_by",
 			"fieldtype": "Select",
-			"options": ["Customer Group", "Customer", "Item Group", "Item", "Territory", "Invoice"]
```

Comment:
Why are we removing the options ?

Reply:
as per the DOD, it's mentioned to keep just customer and item in the options!.